### ISSUE TITLE
Report selection input is missing a label

### ISSUE THREAD

**Issue by vpicone** [WoWAnalyzer/WoWAnalyzer#3642]
For accessibility purposes, input elements should have labels associated with them. This allows screen readers to properly announce the purpose of an input.

ref:
- [WCAG](https://www.w3.org/WAI/tutorials/forms/labels/)
- [Bootstrap docs](https://getbootstrap.com/docs/3.3/components/#callout-inputgroup-form-labels)

**Comment by MartijnHols** [comment]
Thanks for your input. In general I agree with you, however in this instance I think omitting the label makes the design cleaner for users without missing clarity. We already have the introductory text explain what should be inserted in the input, as well as a placeholder to indicate the format. This should be sufficient to explain the intent of the input, even for screen readers.

Considering this is a fan site for a video game, I think we can be somewhat selective in how accessible we make it.

**Comment by vpicone** [comment]
> I think we can be somewhat selective in how accessible we make it.

Strong disagree with this premise, Blizzard takes a lot of steps to accommodate disabled players. Not sure why we wouldn't want to match that effort.

If you click that bootstrap link, you'll see they suggest you should always add a label for this component. Placeholders are not visible to screen readers and shouldn't be used to convey instructions like format.

Regardless, I respect your decision. Maybe someone will revisit in the future.

**Comment by MartijnHols** [comment]
I reckon that is a bug in screen readers that should be fixed instead. However I think we should be able to accommodate your feedback by adding an aria-labeledby on the input pointing to the text underneath the header.
